README.md

```diff
@@ -0,0 +1,32 @@
+Qt5 OpenEmbedded/Yocto Project layer
+====================================
+
+This layer depends on:
+
+URI: git://github.com/openembedded/oe-core.git
+branch: master
+revision: HEAD
+
+When building stuff like qtdeclarative, qtquick, qtwebkit, make sure that
+you have required PACKAGECONFIG options enabled in qtbase build, see qtbase.inc
+for detail.
+
+
+Contributing
+------------
+
+Please submit any patches against the meta-qt5 layer by using the GitHub
+pull-request feature.  Fork the repo, make a branch, do the work, pull from
```

Comment:
Should we say "rebase on latest target branch in upstream" instead of just pull from upstream?

Reply:
Ok, I made a small change and will update.